docs/development/debugging.md

```diff
@@ -0,0 +1,65 @@
+(debugging)=
+# Interactive Debugging
+
```

Comment:
It would be good to also link to https://emscripten.org/docs/porting/Debugging.html in the introduction